ivy/functional/frontends/torch/other_operations .py

```diff
@@ -0,0 +1,4 @@
+import ivy
+# function for bincount
+def bincount(input, weights=None, minlength=0):
+    return ivy.bincount(input, weights=weights, minlength=minlength)
```

Comment:
the functional API doesn't have a `bincount` function you'll need to implement it compositionally

Reply:
or an example?